Blaise-Shyaka/AnnounceIT
# #170842443 Change advertisement status API endpoint

# What does this PR do?

This PR enables an admin to change the status of an advertiser

# Description of the task to be completed

- Create a PATCH /announcement/:id/:status endpoint
- Add authentication
- Write tests for that endpoint 

# How should this be manually tested?

- Clone this repository
- Type in your terminal "npm run test"
- 18 tests should pass

# What are the relevant pivotal tracker stories?

#170842443
